js/utilities.js

```diff
@@ -2948,6 +2948,50 @@
         return unescape(module._fixedEncodeURIComponent(JSON.stringify(obj)));
     };
 
+    module._versionDecodeBase32 = function (version) {
```

Comment:
Please just add the jsdoc to explain what this function is doing. You don't need to explain the whole algorithm just a simple line of saying what is the input and what the output is going to be.